.github/workflows/workflow.yml

```diff
@@ -0,0 +1,81 @@
+name: Main workflow
+
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: 0 0 * * 5
+
+jobs:
+  plugin_test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
```

Comment:
This does support windows. Does github actions not have that as an option?

Reply:
Ah indeed.